client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -1,114 +1,127 @@
-      </div>
-  </div>
-
-  <button id="btnCrop" class="pull-right" ng-click="applyCrop()" ng-if="metadata.crop_sizes" translate>APPLY CROPS</button>
-  <div ng-if="item.type == 'audio'">
-    <audio controls="controls" sd-sources data-renditions="item.renditions"></audio>
-  </div>
-
-  <div ng-if="item.type == 'video'">
-    <video controls="controls" sd-sources data-renditions="item.renditions"></video>
-  </div>
-</div>
-
-<div class="composite-preview" ng-if="item.type === 'composite'">
-  <div sd-package-items-edit ng-model="item.groups" ng-if="_editable"></div>
-  <div sd-package data-item="item" class="composite-preview" ng-if="!_editable"></div>
-</div>
-
-<div class="field pull-left" ng-if="item.sign_off">
-    <label translate>Sign-Off</label>
-    <div>{{item.sign_off}}</div>
-</div>
\ No newline at end of file
+<div class="field" ng-class="{'limit-error': item.headline.length > limits.headline}">
+  <label translate>Headline</label>
+    <span sd-character-count data-item="item.headline" data-html="true" data-limit="limits.headline"></span>
+    <div id="title" class="headline"
+        sd-text-editor
+        data-config="{disableToolbar: true, disableReturn: true}"
+        data-language="item.language"
+        ng-model="item.headline"
+        ng-change="autosave(item)"
+        ng-if="_editable"
+        ng-trim="false"></div>
+  <div class="headline" ng-if="!_editable">{{item.headline}}</div>
+</div>
+
+<div class="field" class="abstract" ng-class="{'limit-error': item.abstract.length > limits.abstract}">
+  <label translate>Abstract</label>
+    <span sd-character-count data-item="item.abstract" data-html="true" data-limit="limits.abstract"></span>
+    <div id="abstract" class="abstract"
+        sd-text-editor
+        data-config="{disableToolbar: true}"
+        data-language="item.language"
+        ng-model="item.abstract"
+        ng-change="autosave(item)"
+        ng-if="_editable"
+        ng-trim="false"></div>
+  <div class="abstract" ng-if="!_editable">{{item.abstract}}</div>
+</div>
+
+<div class="field">
+  <label translate>By</label>
+  <input type="text"
+         class="byline"
+         id="byline"
+         ng-model="item.byline"
+         ng-change="autosave(item)"
+         ng-if="_editable">
+  <div ng-if="!_editable">{{item.byline}}</div>
+</div>
+
+<div class="field">
+    <label translate>Dateline</label>
+    <div class="dateline">
+        <div sd-meta-locators
+             class="dateline-city"
+             ng-show="_editable"
+             data-list="metadata.cities"
+             data-item="item"
+             data-fieldprefix="dateline"
+             data-field="located"
+             data-postprocessing="updateDateline(item, city)"
+             data-change="autosave(item)">
+        </div>
+
+        <div class="dateline-date">
+            <select id="datelineMonth"
+                    ng-options="monthValue as monthName for (monthName, monthValue) in monthNames track by monthName"

```

Comment:
@pavlovicnemanja  Actually, there is a potential issue here. If you look at the [ng-options docs](https://code.angularjs.org/1.4.6/docs/api/ng/directive/ngOptions), it says that `select as` and `track by` are not  designed to work together. `track by` should thus be omitted here.

BTW, `monthValue as` is also redundant, because if `select as` part is omitted, it defaults to `value` (which `monthValue` in this case).